src/pages/billing/List.js

```diff
@@ -283,6 +315,10 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
     setIsAlphaSpendReportUser(await Ajax(signal).Groups.group(getConfig().alphaSpendReportGroup).isMember())
   })
 
+  const loadAlphaAzureMember = reportErrorAndRethrow('Error loading azure alpha group membership')(async () => {
+    setIsAlphaAzureUser(await Ajax(signal).Groups.group(getConfig().alphaAzureGroup).isMember())
+  })
```

Comment:
I sometimes think the promise methods can make the flow more clear. Your call:
```suggestion
  const loadAlphaAzureMember = reportErrorAndRethrow('Error loading azure alpha group membership')(() =>
    Ajax(signal).Groups.group(getConfig().alphaAzureGroup).isMember()
    .then(setIsAlphaAzureUser)
  )
```

Reply:
I tend to forget this way of writing the code, but I do think it's somewhat cleaner. I'll switch it.

Reply:
This lead to some lint warnings which I'm sure I can fix, but I think I'm going to stick with the async/await syntax.